db/deploy/Dockerfile

```diff
@@ -8,7 +8,37 @@ ENV LIQUIBASE_VERSION=4.26.0
 ENV MARIADB_JDBC_VERSION=3.0.3
 
 # Install system dependencies
-RUN apt-get update && apt-get install -y --no-install-recommends gcc git ssh default-jdk wget unzip
+RUN apt-get update && apt-get install -y --no-install-recommends gcc git ssh default-jdk wget unzip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Create a non-root user and group
+RUN groupadd -r appuser && useradd -r -g appuser -d /home/appuser -m -s /bin/bash appuser
+
+# Download and install Liquibase
+RUN wget https://github.com/liquibase/liquibase/releases/download/v${LIQUIBASE_VERSION}/liquibase-${LIQUIBASE_VERSION}.zip \
+    && unzip liquibase-${LIQUIBASE_VERSION}.zip -d /opt/liquibase \
+    && chmod +x /opt/liquibase/liquibase \
+    && rm liquibase-${LIQUIBASE_VERSION}.zip \
+    # Clean up to reduce layer size
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Change ownership of the liquibase directory to the non-root user
+RUN chown -R appuser:appuser /opt/liquibase
+
+# Switch to the non-root user
+USER appuser
+
+# Set the working directory
+WORKDIR /home/appuser
+
+# Add any additional commands here, such as copying application files, installing Python dependencies, etc.
+# COPY requirements.txt .
+# RUN pip install -r requirements.txt
+
+# Entry point or command to run your application
+# ENTRYPOINT ["python", "your_script.py"]
 
 # Download and install Liquibase
 RUN wget https://github.com/liquibase/liquibase/releases/download/v${LIQUIBASE_VERSION}/liquibase-${LIQUIBASE_VERSION}.zip \
```

Comment:
Is this duplicating wget at LN 19?

Reply:
Ah of course it is. I'll remove the dup. Thanks for that pick up.